react/features/prejoin/functions.js

```diff
@@ -0,0 +1,194 @@
+// @flow
+
+/**
+ * Selector for the visibility of the 'join by phone' buttons.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {boolean}
+ */
+export function areJoinByPhoneButtonsVisible(state: Object): boolean {
+    return state['features/prejoin'].buttonsVisible;
```

Comment:
No entity sets this as of now, this is expected, right? Can we make the attribute more explicit? "buttonsVisible" is not very specific as there may be many buttons.

Reply:
This will be fixed in my next MR. Buttons visibility depends on the fact that there exists a url for the _join by phone_. Having to to default to an undefined prop is intended.